AMI build fails during "make prep-tools"

```
ESC[0;32m    amazon-ebs: which go-junit-report || go get github.com/jstemmer/go-junit-reportESC[0m
ESC[1;31m==> amazon-ebs: which: no go-junit-report in (/usr/local/bin:/usr/bin:/home/openshift/bin:/home/openshift/go/bin)ESC[0m
ESC[1;31m==> amazon-ebs: go: github.com/openshift/api@v3.9.0+incompatible: reading github.com/openshift/api/go.mod at revision v3.9.0: unknown revision v3.9.0ESC[0m
ESC[1;31m==> amazon-ebs: make: *** [prep-tools] Error 1ESC[0m
2021/01/04 01:02:22 packer-builder-amazon-ebs plugin: [ERROR] Remote command exited with '2': /tmp/script_9696.sh  --branch master
2021/01/04 01:02:22 packer-builder-amazon-ebs plugin: [INFO] RPC endpoint: Communicator ended with: 2
```

I'm going to look into this ... 

FYI @clemensutschig @georgfedermann

--- a/tests/go.mod
+++ b/tests/go.mod
@@ -15,7 +15,7 @@ require (
 	github.com/jstemmer/go-junit-report v0.9.1
 	github.com/kr/pretty v0.2.1 // indirect
 	github.com/modern-go/reflect2 v1.0.1 // indirect
-	github.com/openshift/api v3.9.0+incompatible
+	github.com/openshift/api v0.0.0-20180801171038-322a19404e37
 	github.com/openshift/client-go v3.9.0+incompatible
 	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
 	github.com/spf13/pflag v1.0.5 // indirect

